src/openepd/model/base.py

```diff
@@ -18,6 +18,8 @@
 import json
 from typing import Any, Callable, Generic, Optional, Type, TypeAlias, TypeVar
 
+from pydantic.v1.main import ModelMetaclass
```

Comment:
Is this correct? I mean, you are not using compatibility layer.

Reply:
Good point!